tests/unit/test_published_clusters.py

```diff
@@ -1,3 +1,5 @@
+from functools import partial
+import json
 from unittest import TestCase
```

Comment:
Note: In the future use [pytest fixtures](https://docs.pytest.org/en/latest/fixture.html) instead of `unittest.TestCase`